HW14/homework_14.py

```diff
@@ -0,0 +1,111 @@
+import json
+import time
+import re
+from datetime import datetime
+
+
+def name_time_decorator(func):
```

Comment:
Was it the task to create a decorator?

Reply:
Decorator is a part of contact book app. It make a log of each "read/write" function in app.